api/tests/unit/worker_test.js

```diff
@@ -7,139 +7,91 @@ import { UserAnonymizedEventLoggingJobController } from '../../src/shared/applic
 import { JobGroup } from '../../src/shared/application/jobs/job-controller.js';
 import { config } from '../../src/shared/config.js';
 import { JobExpireIn } from '../../src/shared/infrastructure/repositories/jobs/job-repository.js';
-import { registerJobs } from '../../worker.js';
+import { registerJobs, startPgBoss } from '../../worker.js';
```

Comment:
Le diff est difficile à lire.
En résumé, ajout de 2 tests : 

- should not register jobs when pgboss is null
- #startPgBoss